package.json

```diff
@@ -1,42 +1,62 @@
 {
   "name": "taskcluster-client",
-  "version": "1.2.1",
+  "version": "2.0.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
   "description": "Client for interfacing taskcluster components",
   "license": "MPL-2.0",
+  "files": [
+    "build",
+    "apis.json"
+  ],
+  "main": "build/index.js",
+  "browser": "build/browser.js",
   "scripts": {
-    "test": "./test/runtests.sh"
-  },
-  "browser": "./browser.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/taskcluster/taskcluster-client.git"
+    "build:browser": "webpack",
+    "build:node": "babel src --out-dir build --source-maps inline --ignore node_modules",
+    "build": "npm run build:node && npm run build:browser",
```

Comment:
These build scripts are only there for finer control, the only real build command you need is `npm run build`. This would be the same whether you used babel-compile or not.